README.adoc

```diff
@@ -147,15 +142,14 @@ This annotation indicates a dependency injection.
 You are injecting your `InventoryManager` bean into the `InventoryResource` class.
 This injects the bean in its specified context (application-scoped) and makes all of its functionalities
 available without the need of instantiating it yourself.
-The injected bean can then be invoked directly via the `manager.get(hostname)`
+The injected bean can then be invoked directly through the `manager.get(hostname)`
 and `manager.list()` function calls.
 
```

Comment:
changed via to make it easier if ever translated...

Reply:
Ok. There's another via in this guide, is that a problem too?

Reply:
yes--can present problems in the future. (Thought I caught them all, will go an edit.)